script/Arcueid/Day 4/04_00_ARC04_6_2C.txt

```diff
@@ -358,37 +388,41 @@ This foreigner... no, this foreign vampire knows quite a lot about Japanese cult
 [sha:d861bd5e5c83c555f904faf883605bdfb88b6728]{
 -- Page 558, Offset 7072.
 -- 長い年月を生きた吸血鬼ほど、崩壊する肉体の補完は通常の命では間に合わない。だから力のある獣を取りこんで、それを体として使っている。
-The longer a vampire lives, the harder it is for ordinary cells to make up for their decaying body. That's why they capture a powerful beast and use it as their body.
+The longer a vampire lives, the harder it is for ordinary cells to make up for their decaying body.
+ That's why they capture a powerful beast and use it to supplement their body.
```

Comment:
Don't really like the second sentence, will come back to it later.

Reply:
don't mind the move to plural, but "When necessary, each is returned to its original state and put to work" would be cleaner. That version sounds like the _Vampire_ is put into their original state.